actions.go

```diff
@@ -141,6 +141,7 @@ type EmailNotifier struct {
 	Host     string
 	From     string
 	To       string
+	TlsPort  string
```

Comment:
It would be TLSPort or tlsPort according to this: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

I will update to TLSPort for consistency with the other CamelCase options.